todo-app/src/components/TodoList.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { inject, observer } from "mobx-react";
+import ListItem from "./ListItem";
+
+
+@inject("todoItems")
+@observer    
+export default class TodoList extends React.Component {
+
+    render() {
+        const { todoItems } = this.props;
```

Comment:
You are destructuring twice for no reason. You can just do:

    const { todos, doneCount } = this.props.todoItems